extensions/windows-event-log/tests/CMakeLists.txt

```diff
@@ -17,17 +17,17 @@
 # under the License.
 #
 
-set(WEL_INTEGRATION_TESTS  "BookmarkTests.cpp" "ConsumeWindowsEventLogTests.cpp" "MetadataWalkerTests.cpp")
+set(WEL_TESTS  "BookmarkTests.cpp" "ConsumeWindowsEventLogTests.cpp" "LookupCacherTests.cpp" "MetadataWalkerTests.cpp")
 if (TEST_CUSTOM_WEL_PROVIDER)
     execute_process(COMMAND
         "${CMAKE_CURRENT_LIST_DIR}/custom-provider/generate-and-register.bat"
         "${CMAKE_CURRENT_LIST_DIR}/custom-provider"
     )
-    list(APPEND WEL_INTEGRATION_TESTS "CWELCustomProviderTests.cpp")
+    list(APPEND WEL_TESTS "CWELCustomProviderTests.cpp")
```

Comment:
The problem is that `generate-and-register.bat` can only be run as Administrator because of `wevtutil`, and we don't want the test runner to require Administrator rights by default.

I'm not sure if it is possible to run a GitHub action as Administrator; if it is, then we could do that.  I have create a Jira for this: https://issues.apache.org/jira/browse/MINIFICPP-2046.